pages/contribute.md

```diff
@@ -0,0 +1,63 @@
+
+MDAnalysis maintains **three GitHub repositories**, each dedicated to a specific area of contribution:
+
+- [Main Code Repository]({{ site.github.repo }}): for code development and API documentation.
+- [User Guide Repository]({{ site.docs.userguide.repo }}): containing installation instructions, tutorials and learning resources.
+- [Website Repository][]: Hosts the website and blog.
+
+---
+
+## Contributing to the Code
+
+If you're interested in developing new features, fixing bugs, or improving performance, contributing to the MDAnalysis codebase is a great way to get involved.
+
+To get started:
+- Read the [Contributing to the Main Codebase]({{ site.docs.userguide.url }}/stable/contributing_code.html#working-with-mdanalysis-code) guide, which outlines all the necessary steps.
+- Browse [GitHub Issues]({{ site.github.issues }}) to find open tasks (look for "good first issue" if you're a beginner). You can also report bugs or suggest improvements there.
+- Follow best practices, including [writing and running tests]({{ site.docs.userguide.url }}/stable/testing.html#testing) and [updating the documentation]({{ site.docs.userguide.url }}/stable/contributing_code.html#building-code-documentation) for any changes.
+
+---
+
+## Contributing to Documentation
+
+Good documentation is essential to making MDAnalysis accessible to users and developers. MDAnalysis maintains two types of documentation: the [User Guide]({{ site.docs.userguide.url }}) and the [API Reference]({{ site.docs.mdanalysis.url }}/stable/index.html).
+
+### User Guide
+
+The [User Guide]({{ site.docs.userguide.url }}/stable/index.html) provides tutorials and explanatory content to help users install and learn MDAnalysis. You can contribute by updating unclear explanations or adding new tutorials or examples. 
+
+To get started:
+- Follow the [Contributing to the User Guide]({{ site.docs.userguide.url }}/stable/contributing_docs.html).
+- Check for open documentation-related issues in the [User Guide GitHub Issues]({{ site.docs.userguide.issues }}).
+
+### API Reference
+
+The [API reference]({{ site.docs.mdanalysis.url }}/stable/index.html) is **automatically generated** from *docstrings* in the codebase and serves as a reference for developers. You can contribute by improving *docstrings* for functions, classes, and modules or updating outdated documentation.
+
+To get started:
+- Follow the [API Documentation Contribution Guide]({{ site.docs.userguide.url }}/stable/contributing_code.html#working-with-mdanalysis-docs).
+- Browse open API documentation issues in [GitHub Issues]({{ site.github.issues }}).
+
+---
+
+## Other Contributions
+
+If you are interested in contributing in other ways, such as improving the website or blog, you can refer to the [README file][] in the [Website Repository][]. While the website is already maintained by the MDAnalysis team, minor fixes and updates are always welcome. 
```

Comment:
Mentioning the website here is ok, just no need to advertise it at the same level as core library, user guide, other projects, and MDAKits.